autoload/committia/git.vim

```diff
@@ -111,7 +111,10 @@ endfunction
 
 function! s:execute_git(cmd) abort
     try
+        let l:shellslash = &shellslash
+        set shellslash
         let [git_dir, work_tree] = s:search_git_dir_and_work_tree()
+        let &shellslash = l:shellslash
```

Comment:
This does not restore `shellslash` when `s:search_git_dir_and_work_tree()` throws exception.

```vim
let l:shellslash = &shellslash
try
    set shellslash
    let [git_dir, work_tree] = s:search_git_dir_and_work_tree()
catch
    throw 'committia: git: Failed to retrieve git-dir or work-tree: ' . v:exception
finally
    let &shellslash = l:shellslash
endtry
```

Reply:
Great! let me fix this

Reply:
fixed, please have a look